### DISCUSSION TITLE
Be able to enter custom bedrock Model IDs

### DISCUSSION THREAD

**Discussion by JacoMoolman** [cline/cline#649] [Feature Requests]
Can we have an option to enter the custom bedrock model to use. I have access to a specific model and version on bedrock that is not available in the pull down model menu. 

EG:
anthropic.claude-3-sonnet-20240229-v1:0

That is not available in in the model options in the settings for bedrock. 

![image](https://github.com/user-attachments/assets/763018be-223d-458c-89da-866f5e8dbcd0)

**Comment by SebastianAwaze** [comment]
Yes, I would also like to be able to do the same. This will be needed especially with fine tuned agents.
